package.json

```diff
@@ -102,12 +102,16 @@
     "sinon": "9.2.1",
     "subscriptions-transport-ws": "0.9.18",
     "supertest": "6.0.1",
-    "ts-node": "9.0.0",
-    "typescript": "4.1.2",
+    "ts-node": "10.0.0",
+    "typescript": "4.3.2",
```

Comment:
Since we upgraded TypeScript, I think we should also upgrade `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` to the latest version (`4.26.0`) to avoid the warning:
```
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
```

At the same time, it would make sense to upgrade `eslint` to the latest `7.27.0`. :+1: